introduction.md

```diff
@@ -1,3 +1,9 @@
 Hello, I am jdip!
 
-Favorite codeschool path is JavaScript.
\ No newline at end of file
+Favorite codeschool path is JavaScript.
+
+Deadly Skills:
+
+* Git
+* Javascript
+* Python
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?